core/lib/config/src/configs/contracts.rs

```diff
@@ -83,6 +85,8 @@ impl ContractsConfig {
             governance_addr: Address::repeat_byte(0x13),
             prover_at_genesis: ProverAtGenesis::Fri,
             snark_wrapper_vk_hash: H256::repeat_byte(0x09),
+            base_token_addr: Address::from_str("0x0000000000000000000000000000000000000001")
+                .unwrap(),
```

Comment:
Nit: Given that we use fake addresses here anyways, I think it'd be fine to use `Address::repeat_byte(0x14)`, this way you'll avoid having to import `FromStr` and doing an `unwrap`.

Reply:
Done